Makefile

```
@@ -24,9 +25,16 @@ etcd-release: ## Interactively starts the release workflow.
 ##@ Helm dev targets - The etcd needs a copy of the targets from k8s.mk without image-import because we use a external image here.
 
 .PHONY: k8s-helm-etcd-apply
-k8s-helm-etcd-apply: ${BINARY_HELM} k8s-helm-generate $(K8S_POST_GENERATE_TARGETS) ## Generates and installs the helm chart.
+k8s-helm-etcd-apply: ${BINARY_HELM} helm-generate $(K8S_POST_GENERATE_TARGETS) ## Generates and installs the helm chart.
```

```suggestion
helm-etcd-apply: ${BINARY_HELM} helm-generate $(K8S_POST_GENERATE_TARGETS) ## Generates and installs the helm chart.
```
We could rename the targets to match the new naming-pattern from k8s-component.mk